pkgs/by-name/li/librepods/package.nix

```
@@ -0,0 +1,52 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  cmake,
+  kdePackages,
+  qt6,
+  pulseaudio,
+}:
+stdenv.mkDerivation (finalAttrs: {
+  pname = "librepods";
+  version = "0.1.0-rc.4-unstable-2025-09-02";
+
+  src = fetchFromGitHub {
+    owner = "kavishdevar";
+    repo = finalAttrs.pname;
```

```suggestion
    repo = "librepods";
```
relying on `pname` matching repo name isn't reliable, see Catppuccin themes as counterexamples